cabal.project

```diff
@@ -159,8 +159,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-node
-  tag: 43393ac88faa99986ecaa0be409e8b6353f5e9fe
-  --sha256: 0g2m1gj0jqsk8gwx19n1h6kh3xjywwvxzw5caqj4jwz5m2z1cmml
+  tag: 497e05522a5d909d671db4427f721f7d888ee4b4
+  --sha256: 1jdz2xpva232vi62662za5zvgs3y6p50fvgj17iqpn0dvv9n1dwb
```

Comment:
see https://github.com/input-output-hk/cardano-node/pull/4518

Reply:
This is following the `backport-purity-1.35.0-rc4` branch. In the future we can follow master again as the PR was merged.

Reply:
Removed as the node PR was merged so this is no longer needed.